### PR TITLE
[16.0][MIG] fieldservice_geoengine: Migration to 16.0 

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 16.0

cc https://github.com/APSL 153662

@miquelalzanillas @lbarry-apsl @javierobcn @mpascuall please review

Issue: https://github.com/OCA/field-service/issues/963

### CODE REVIEW

**Comment by peluko00** [comment]
Hi @max3903, the query for base_geoengine on postgis are failing because the version of the postgres is 12 and needs the 13-3.4.

**Comment by max3903** [comment]
@peluko00 Can you update it?

https://github.com/APSL/field-service/blob/16.0-mig-fieldservice_geoengine/.github/workflows/test.yml#L45

**Comment by peluko00** [comment]
> @peluko00 Can you update it?
> 
> https://github.com/APSL/field-service/blob/16.0-mig-fieldservice_geoengine/.github/workflows/test.yml#L45

Solve it! Please review when you can

**Comment by max3903** [comment]
@peluko00 Can you explain why you removed the test?

**Comment by peluko00** [comment]
> @peluko00 Can you explain why you removed the test?

I put again

File: fieldservice_geoengine/models/fsm_location.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+class FSMLocation(models.Model):
+    _inherit = "fsm.location"
+
+    # Geometry Field
+    shape = fields.GeoPoint("Coordinate", compute="_compute_shape", store=True)
+    stage_name = fields.Char(related="stage_id.name", string="Stage Name")
+    custom_color = fields.Char(related="stage_id.custom_color", string="Stage Color")
+
+    @api.model
```
Comment (blounts) [inline fieldservice_geoengine/models/fsm_location.py]: Thanks for the work on this... 
2024-05-12 03:17:01,463 21891 WARNING odoo16demo odoo.api.create: The model odoo.addons.fieldservice_geoengine.models.fsm_location is not overriding the create method in batch

**Comment by legacyplumbing** [comment]
This is the only warning that I got when applying this pr... 
2024-05-12 03:17:01,463 21891 WARNING odoo16demo odoo.api.create: The model odoo.addons.fieldservice_geoengine.models.fsm_location is not overriding the create method in batch

File: fieldservice_geoengine/models/fsm_location.py (hunk 1 above)
Comment (peluko00) [reply fieldservice_geoengine/models/fsm_location.py]: Fix it, many thanks!

**Review by max3903 — APPROVED** [review]

**Comment by peluko00** [comment]
Can we merge this? I'm interested to migrate this module to v17 @max3903

**Pull request merged**